apps/fluent-tester/src/TestComponents/CheckboxExperimental/CheckboxTest.tsx

```diff
@@ -147,15 +160,23 @@ const TokenCheckbox: React.FunctionComponent = () => {
   );
 };
 
+const UnSupportedOnAndroid = () => {
+  return (
+    <Text style={{ margin: 10, fontSize: 14 }} color="red">
+      Unsupported on Android
+    </Text>
+  );
+};
+
 const checkboxSections: TestSection[] = [
   {
     name: 'Basic Checkboxes',
     testID: EXPERIMENTAL_CHECKBOX_TESTPAGE,
-    component: BasicCheckbox,
+    component: Platform.OS === 'android' ? AndroidBasicCheckbox : BasicCheckbox,
   },
   {
     name: 'Size Checkboxes',
-    component: SizeCheckbox,
+    component: Platform.OS === 'android' ? UnSupportedOnAndroid : SizeCheckbox,
```

Comment:
I'm not a fan of this approach for a few reasons:

1) Adding conditional logic for a specific platform IMO should be done sparingly. The check becomes more difficult as more platforms are added and it's harder to keep track of what's happening where

2) This isn't exclusive to Android. Other platforms like iOS and macOS may or do not support different sizes and shapes for their checkboxes and I'd like a better way to convey that than "unsupported"

Reply:
Correct. 
Addressed on https://github.com/microsoft/fluentui-react-native/pull/2416